sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/SchedulerApiServerTest.java

```diff
@@ -28,7 +28,7 @@ public void testApiServerReady() throws Exception {
         SchedulerApiServer schedulerApiServer = new SchedulerApiServer(12345, Collections.emptyList(),
                 Duration.ofMillis(TIMEOUT_MILLIS));
         new Thread(schedulerApiServer).start();
-        Thread.sleep(TIMEOUT_MILLIS * 10);
```

Comment:
You mean the `ready()` method? That's the method that causes the unexpected termination due to a timeout when `assertTrue(schedulerApiServer.ready());` is called on the line following the sleep.

A better solution to this may be to rather increase the timeout passed to the constrcutor on line 28. What are your thoughts on this @benclarkwood?